core/src/lib.rs

```diff
@@ -93,6 +93,16 @@ pub fn get_api() -> &'static GodotApi {
     unsafe { GODOT_API.as_ref().expect("API not bound") }
 }
 
+#[inline]
+#[doc(hidden)]
+#[cfg(feature = "api_struct_raw_init")]
```

Comment:
Not very important, but for what it's worth I wouldn't mind including this without a feature flag. I don't think having it available by default adds any build/size/runtime cost.
Your call.